docs/crawler.js

```diff
@@ -0,0 +1,103 @@
+// This is the code example for the crawler configuration file for https://crawler.algolia.com/
+new Crawler({
+    rateLimit: 8,
+    maxDepth: 10,
+    maxUrls: 5000,
+    startUrls: ["https://docs.robusta.dev/master/"],
+    renderJavaScript: false,
+    sitemaps: [],
+    ignoreCanonicalTo: false,
+    discoveryPatterns: ["https://docs.robusta.dev/master/**"],
+    schedule: "at 13:11 on Thursday",
+    actions: [
+      {
+        indexName: "robusta",
+        pathsToMatch: ["https://docs.robusta.dev/master/**"],
+        recordExtractor: ({ helpers, url }) => {
+          return helpers.docsearch({
+            recordProps: {
+              lvl0: {
+                selectors: ["article h1", "head > title"],
+                defaultValue: "Documentation",
+              },
+              lvl1: ["article h2", "h2"],
+              lvl2: ["article h3", "h3"],
+              lvl3: ["article h4", "h4"],
+              lvl4: ["article h5", "h5"],
+              lvl5: ["article h6", "h6"],
+              content: ["article p, article li, article code"],
+              pageRank: (() => {
+                if (url.pathname.includes("/installation/index.html")) {
+                  return 10;
+                } else if (url.pathname.includes("/triggers/kubernetes.html")) {
```

Comment:
@LeaveMyYard we need to prioritize all trigger and action pages, not just `/triggers/kubernetes.html` and `/actions/event-enrichment.html`